content/docs/guide/gamemodes/climb.md

```diff
@@ -11,16 +11,92 @@ tags:
+Note that since Airstrafing causes the player to deviate from their intended destination (assuming it is straight in front of them), Long Jumps typically incoroporate multiple left and right Airstrafes into a jump which counteract each other and keep the player close to a straight line. These multiple starfes in opposite directions gives the apperance of a "flicky" or "jittery" Airstrafe but a lot of practices goes into syncing the A and D keys with quick but smooth mouse movements.
+
+Sometimes you may not be able to take a direct line due to an obstruction.  
+The 2 most common forms of this is rounding a corner (often without being able to see the next platform) and 2 platforms separated by a thin wall.
+In this case, air strafing is not used to gain speed but as a form of air control. The principle is the same but typically smooth turns that stay close to the wall are needed (don't bump into the wall or else you will take out all your speed and fall straight down!).  
+
+### Bunnyhop (Bhop)
+
+Bhop has reduced potential compared to the Bhop game mode.  
+Bhopping has its own speedcap: **TBD (Momkz)**.      **380 (KZT)**,      **300 (1.6 Climb)**. The default running speed in all Climb modes is 250.  
+This cap is seperate from the **maxvelocity** cap (how fast the engine lets you go before not bothering to give anymore speed 3500 by default).  
+Anytime you jump the Bhop speed cap will be used meaning even if the player gained speed purely through falling or surfing,  
+as soon as they try to jump off the ground with this speed, the cap will proc. If the player reaches this cap and tries to jump again,  
+thier speed will be reset to:  **TBD (MomClimb)**,     **380 (KZT)**,       **240 (1.6 Climb)**. 
+
+However this Bhop cap still allows the player to gain more speed than a Prestrafe hence, Bhopping has the potential to go further than a Long Jump. Useful to remember when dealing with diffcult Long Jumps (although this may form a bad habit if relied upon).  
+Bhop jumps can also be required by the mapper by using "Bhop Blocks" that teleport you back if you try standing on them, same as the Bhop gamemode. Conversely, mappers can also add triggers that disable Bhops inside an area.  
+
+### Surf
+
+Surf is very similar to its game mode counterpart. All Climb modes are 100 tick while the Surf gamemode is 66 tick. This means gaining speed is quicker and losing speed is harder in Climb compared to Surf.  
+Maps made with MomClimb and KZT in mind can have tradtional surf sections. The length and diffculty of these sections is entirely up to the map. Usually they are much shorter (sometimes only 1 ramp) compared to dedicated Surf maps.  
+1.6 Climb has a special version of Surf called **Slide**.  
+1.6 Climb has much lower airacelleration  (10aa compared to MomClimb and KZT's 100aa). This means players can't as easily flick onto a ramp ("board") and genearate speed, or flick off a ramp and generate height.    
```

Comment:
> 1.6 Climb has much lower airacelleration

misspelt acceleration, "air acceleration" should probably be 2 separate words

> ramp ("board") and genearate speed

misspelt "generate"